src/editors/containers/TextEditor/components/SelectImageModal/sortUtils.js

```diff
@@ -0,0 +1,26 @@
+export const OPTIONS = [
```

Comment:
make sure to internationalize the strings you've added based, so we don't have to go back and do them later!